src/bin/parser.js

```diff
@@ -263,7 +250,6 @@ export default class Harmonic {
             metadata.lang = lang;
             metadata.default_lang = config.i18n.default === lang ? false : true;
```

Comment:
Turns out fixing this will break the current version of the JS Rocks theme. I'll open another issue for this.